store/v2alpha1/multi/cache_store.go

```diff
@@ -61,3 +63,13 @@ func (noopCacheStore) Write() {}
 func CommitAsCacheStore(s types.CommitMultiStore) types.CacheMultiStore {
 	return noopCacheStore{newCacheStore(s)}
 }
+
+func (cs *cacheStore) wrapTraceListen(store types.KVStore, skey types.StoreKey) types.KVStore {
```

Comment:
I see this is called on line 33 in `(cs *cacheStore) GetKVStore()` but that line isn't added. What implementation of `wrapTraceListen` was called before this method was added?

Reply:
The inherited implementation in `multi/store.go` was called before but since I modified that and the modified version isn't appropriate for `cacheStore` because there is no notion of a `subStore`, I added the previous implementation of `wrapTraceListen` here